tools/sort.py

```diff
@@ -331,6 +332,7 @@ def sort_face_cnn(self):
         return img_list
 
     def sort_face_cnn_dissim(self):
+        import_face_recognition()
```

Comment:
I can't see that face_recognition is used in this method. Just FaceLandmarksExtractor